config/doctrine.php

```diff
@@ -143,6 +143,8 @@
     |--------------------------------------------------------------------------
     */
     'logger'                     => env('DOCTRINE_LOGGER', false),
+    'debugbar_widget_key'        => env('DOCTRINE_DEBUGBAR_KEY', 'queries'),
```

Comment:
Don't think it's really needed to have a env variable for this. You would change this setting for your project, not per environment.